src/identity/heading-1.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { FC } from 'react';
 
-type Props = {
+export type Heading1Props = {
```

Comment:
Any reason why you would want to export these props? As far as I can see they are not needed anywhere else and only belong to the component.